src/header.ts

```diff
@@ -82,6 +85,20 @@ export interface HeaderConfig {
    */
   titleLimit?: number;
 
+  /**
+   * The orthogonal offset in pixels by which to displace the title from its position along the edge of the chart.
+   *
+   * __Default value:__ `10`, indicating no limit
+   */
+  titleOffset?: number;
+
+  /**
+   * Title orientation (`"top"`, `"bottom"`, `"left"`, or `"right"`).
+   *
+   * __Default value:__ `"left"` for rows and `"top"` for columns.
+   */
+  titleOrient?: TitleOrient;
```

Comment:
Does mapping title orient to orient of underlying title work?

Don't you have to do much more (correctly applying to header/footer, etc.)?

Reply:
Done